### PR TITLE
Initial support of DataFrame.GroupBy by single literal column

### PR DESCRIPTION
**Pull request by kozlov-alexey**

### CODE REVIEW

File: sdc/tests/test_groupby.py [hunk 1]
```diff
@@ -53,6 +54,272 @@
...
+        df = pd.DataFrame({
+                    'A': [2, 1, 1, 1, 2, 2, 1, 0, 3, 1, 3],
+                    'B': np.arange(n, dtype=np.intp)
+        })
+        result = hpat_func(df)
+        result_ref = test_impl(df)
+        pd.testing.assert_frame_equal(result, result_ref)
+
+    @skip_sdc_jit('Fails with old-pipeline from the start')
+    def test_dataframe_groupby_count_by_int(self):
```
Comment (densmirn) [inline sdc/tests/test_groupby.py]: Maybe to combine tests `test_dataframe_groupby_count_by_*` via `subTest`?

File: sdc/tests/test_groupby.py [hunk 2]
```diff
@@ -53,6 +54,272 @@
...
+            return df.groupby('A').max()
+        hpat_func = self.jit(test_impl)
+
+        n = 11
+        df = pd.DataFrame({
+                    'A': [2, 1, 2, 1, 2, 2, 1, 0, 3, 1, 3],
+                    'B': np.arange(n, dtype=np.intp),
+                    'C': np.arange(n, dtype=np.float_),
+                    'D': [np.nan, 2., -1.3, np.nan, 3.5, 0, 10, 0.42, np.nan, -2.5, 23],
+                    'E': [np.inf, 2., -1.3, -np.inf, 3.5, 0, 10, 0.42, np.nan, -2.5, 23]
```
Comment (densmirn) [inline sdc/tests/test_groupby.py]: The data is used in a lot of places. Wouldn't you like to move it to some common place?

**Comment by AlexanderKalistratov** [comment]
@kozlov-alexey conflicts.
Let's merge it as-is for now

File: sdc/tests/test_groupby.py (hunk 1 above)
Comment (kozlov-alexey) [reply sdc/tests/test_groupby.py]: Done!

File: sdc/tests/test_groupby.py (hunk 2 above)
Comment (kozlov-alexey) [reply sdc/tests/test_groupby.py]: Yes, you're probably right, I moved it to a global var.

**Pull request merged**